core/src/labels/labels.cpp

```diff
@@ -58,15 +58,17 @@ void Labels::processLabelUpdate(const ViewState& viewState,
         ScreenTransform transform { m_transforms, transformRange };
 
         // Use extendedBounds when labels take part in collision detection.
-        auto bounds = (onlyTransitions || !label->canOcclude())
+        auto bounds = (onlyRender || !label->canOcclude())
             ? screenBounds
             : extendedBounds;
 
         if (!label->update(mvp, viewState, &bounds, transform)) {
             continue;
         }
 
-        if (onlyTransitions) {
+        if (label->forceInvisible()) { label->occlude(); }
```

Comment:
Hmm ... but I still have to mark the label/marker to be occluded right? (occluded by force!)